org.opent2t.sample.binaryswitch.superpopular/org.opent2t.sample.binaryswitch.superpopular.json

```diff
@@ -2,21 +2,28 @@
   "id": "http://schemas.microsoft.com/org.opent2t.sample.binaryswitch.superpopular#",
   "$schema": "http://json-schema.org/draft-04/schema#",
   "description": "",
-  "title": "OpenT2T Switch",
+  "title": "OpenT2T SuperPopular Binary Switch",
   "definitions": {
     "org.opent2t.sample.binaryswitch.superpopular": {
       "type": "object",
       "properties": {
         "power": {
-          "$ref": "oic.r.switch.binary.json#/definitions/oic.r.switch.binary"
+          "$ref": "oic.r.switch.binary.json#/definitions/oic.r.switch.binary",
+          "description": "Status of the switch."
         }
       }
     }
   },
   "type": "object",
   "allOf": [
-    { "$ref": "oic.core.json#/definitions/oic.core" },
-    { "$ref": "oic.baseResource.json#/definitions/oic.r.baseresource" },
-    { "$ref": "#/definitions/org.opent2t.sample.binaryswitch.superpopular" }
+    {
+      "$ref": "oic.core.json#/definitions/oic.core"
+    },
+    {
+      "$ref": "oic.baseResource.json#/definitions/oic.r.baseresource"
+    },
+    {
+      "$ref": "#/definitions/org.opent2t.sample.binaryswitch.superpopular"
+    }
```

Comment:
Some of the schemas have this expanded formatting, some have the more compact formatting that was removed from this file. We should be consistent (my vote is for the more compact formatting, but it's up to you).